webpack/components/PaginationRow/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Paginator } from 'patternfly-react';
+
+const defaultPerPageOptions = [5, 10, 15, 25, 50];
+
+const initPagination = (props) => {
+  const pagination = props.pagination || {};
+  const defaultPagination = {
+    page: 1,
+    perPage: 10,
```

Comment:
This should probably be hooked up to the user setting. I am fine with handling this in another PR.